// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`php-src tests Testing throw exception doesn't crash with wrong params, variant 1 1`] = `
Program {
  "children": Array [
    Class {
      "attrGroups": Array [],
      "body": Array [],
      "extends": Name {
        "kind": "name",
        "name": "Exception",
        "resolution": "uqn",
      },
      "implements": null,
      "isAbstract": false,
      "isAnonymous": false,
      "isFinal": false,
      "kind": "class",
      "name": Identifier {
        "kind": "identifier",
        "name": "Hello",
      },
    },
    Throw {
      "kind": "throw",
      "what": New {
        "arguments": Array [
          New {
            "arguments": Array [],
            "kind": "new",
            "what": Name {
              "kind": "name",
              "name": "stdClass",
              "resolution": "uqn",
            },
          },
        ],
        "kind": "new",
        "what": Name {
          "kind": "name",
          "name": "Hello",
          "resolution": "uqn",
        },
      },
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
